envs/monkey_zoo/blackbox/tests/performance/utils/telem_parser.py

```diff
@@ -0,0 +1,114 @@
+import copy
+import json
+import logging
+import sys
+from os import listdir, path
+from typing import List, Dict
+
+from tqdm import tqdm
+
+from envs.monkey_zoo.blackbox.tests.performance.utils.fake_ip_generator import FakeIpGenerator
+from envs.monkey_zoo.blackbox.tests.performance.utils.fake_monkey import FakeMonkey
+
+TELEM_DIR_PATH = './tests/performance/test_telems'
+LOGGER = logging.getLogger(__name__)
+
+
+class TelemParser:
+
+    def __init__(self, multiplier: int):
+        self.multiplier = multiplier
+        self.fake_ip_generator = FakeIpGenerator()
+
+    def multiply_telems(self):
+        telems = TelemParser.get_all_telemetries()
+        telem_contents = [json.loads(telem['content']) for telem in telems]
+        monkeys = self.get_monkeys_from_telems(telem_contents)
+        for i in tqdm(range(self.multiplier), desc="Batch of fabricated telemetries", position=1):
+            for monkey in monkeys:
+                monkey.change_fake_data()
+            fake_telem_batch = copy.deepcopy(telems)
+            TelemParser.fabricate_monkeys_in_telems(fake_telem_batch, monkeys)
+            TelemParser.offset_telem_times(iteration=i, telems=fake_telem_batch)
+            TelemParser.save_teletries_to_files(fake_telem_batch)
```

Comment:
Print a newline after the `tqdm` otherwise other input can get messed up on terminal